client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -335,8 +313,7 @@ export class SeoForm extends React.Component {
 		);
 
 		const conflictedSeoPlugin = siteIsJetpack
-			? // Let's just pick the first one to keep the notice short.
-				this.getConflictingSeoPlugins( activePlugins )[ 0 ]
+			? getConflictingSeoPlugins( activePlugins )[ 0 ] // Pick first one to keep the notice short.
```

Comment:
This was already in production so we'd probably heard about it sooner if it was erroring out. :) As @rodrigoi said, `getConflictingSeoPlugins` always returns an array (well provided that `activePlugins` is one, but if it's not it would break sooner on `filter`), and `[][0]` just returns `undefined`. That is fine for our purposes above since `! undefined === true` and `!! undefined === false`. :smile:

When I originally implemented it I used `head` for the similar reasons/confusion that you pointed out, but it's just faster to use `array[0]` (and `head` returns `undefined` for empty array anyway).

Reply:
There shouldn't be an error, I think it does read nicer if we add another helper like `hasConflictingSeoPlugins`. Internally it can do the same thing.

Reply:
> There shouldn't be an error, I think it does read nicer if we add another helper like hasConflictingSeoPlugins or getFirstConflictingPlugin. Internally it can do the same thing.

Updated the name as suggested in dd40d7aea39fb7a2ab6bf69f101c97240ddc8cc1.